packages/docker-engine/Cargo.toml

```diff
@@ -12,6 +12,13 @@ path = "pkg.rs"
 url = "https://github.com/docker/engine/archive/v19.03.12/engine-19.03.12.tar.gz"
 sha512 = "51632ca8cff03440e3b76ad3e7f06b9c6275c776f4c4f9dc88cd7f5aa9b8aa8cb16ce2b3702426cb93f3714265a6cd28d4ba31612db37199e2915821b9ee8fd4"
 
+# This is a large patch, so we don't want to check it into the repo. Source:
+# https://github.com/docker/engine/commit/44a8e10bfc794dbbc4011a62fa8ba71d948d13cf
```

Comment:
Seeing this in context made me realize something... :)

This patch has an externally accessible URI, so it's not the same case as with k8s.

```
https://github.com/docker/engine/commit/44a8e10bfc794dbbc4011a62fa8ba71d948d13cf.patch
```

Because of that, we can reference that URL in the `url` field below, instead of using a `file:///` URL, to make it clearer to others where the patch comes from.  We don't need this comment then, either, because the source becomes obvious.

I'm also pretty sure you can use the http URL in the `Patch100:` line in the spec file, instead of just the filename, the same way we do for `Source` lines.

Sorry for the wild goose chase on this; I didn't originally know that the patch was public.